clients/web/test/testUtils.js

```diff
@@ -1219,6 +1225,25 @@ $(function () {
 girderTest.startApp = function () {
     var defer = new $.Deferred();
     girderTest.promise.then(function () {
+        /* Track bootstrap transitions using our own test event. */
+        $.support.transition = {end: 'girdertest_transitionend'};
+        $.fn.emulateTransitionEnd = function (duration) {
+            girder._inTransition = true;
+            var $el = this;
+            window.setTimeout(function () {
+                girder._inTransition = false;
+                $($el).trigger($.support.transition.end);
+            }, duration);
+            return this;
+        };
```

Comment:
The original source of this code is described here: https://blog.alexmaccaw.com/css-transitions  While I think this works for our purposes, it does break the original intent of the code that the browser specific event is *always* triggered after a transition.  I wonder if this has side effects elsewhere.

I wonder if a better approach would be to bind to `$.support.transition.end` setting `girder._inTransition = false` and to wrap this function setting `girder._inTransition = true` before calling the original.